### PR TITLE
update (MS) links

### PR DESCRIPTION
**Pull request by Jay-o-Way**
- docs to learn
- removed localizors

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
@dotnet-policy-service agree

File: release-notes/2.2/2.2.0/2.2.0-commits.md [hunk 1]
```diff
@@ -928,7 +928,7 @@ An incredible amount of work went into ASP.NET Core for the 2.2 release (as you
 * [`[cef7dfcdbb]`](https://github.com/aspnet/AspNetCore/commit/cef7dfcdbb) Check connection state in ping Typescript client (#2753)
 * [`[02d1d4b1aa]`](https://github.com/aspnet/AspNetCore/commit/02d1d4b1aa) Adding Javadocs (#2745)
 * [`[9585084258]`](https://github.com/aspnet/AspNetCore/commit/9585084258) Using Routing feature branch versions
-* [`[fc8979aeca]`](https://github.com/aspnet/AspNetCore/commit/fc8979aeca) Update TS docs for docs.microsoft.com generation (#2605)
+* [`[fc8979aeca]`](https://github.com/aspnet/AspNetCore/commit/fc8979aeca) Update TS docs for learn.microsoft.com generation (#2605)
```
Comment (mairaw) [inline release-notes/2.2/2.2.0/2.2.0-commits.md:931]: ditto 

```suggestion
* [`[fc8979aeca]`](https://github.com/aspnet/AspNetCore/commit/fc8979aeca) Update TS docs for docs.microsoft.com generation (#2605)
```

File: release-notes/3.1/3.1.32/3.1.32.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The following repos have been updated.
 | OS | Development Environment | .NET Core SDK |
 | :-- | :-- | :--: |
 | Windows | Visual Studio 2019 version 16.9 | [3.1.426](#downloads) |
-| macOS | Visual Studio for Mac | [Visual Studio for Mac .NET Core Support](https://docs.microsoft.com/visualstudio/mac/net-core-support) |
+| macOS | Visual Studio for Mac | [Visual Studio for Mac .NET Core Support](https://learn.microsoft.com/visualstudio/mac/net-core-support) |
```
Comment (mairaw) [inline release-notes/3.1/3.1.32/3.1.32.md:48]: I would update the link to the last redirect 

```suggestion
| macOS | Visual Studio for Mac | [Visual Studio for Mac .NET Core Support](https://learn.microsoft.com/visualstudio/mac/supported-versions-net) |
```

File: release-notes/6.0/known-issues.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ The optional workload manifest MSIs in the SDK populate the Language column in t
 #### Workaround
 Running the 6.0.101 SDK bundle (without using MU) results in the context changing to MSIINSTALLCONTEXT_MACHINE, this allows the API call to query the INSTALLEDLANGUAGE to complete and the SDK Wix bundle install succeeds.
 
-Therefore a workaround for this issue is to install the 6.0.101 SDK bundle manually by downloading it from the [.NET download site](https://dotnet.microsoft.com/en-us/download/dotnet/6.0). Once this is successfully installed scanning MU again will result in clearing the previous error.
+Therefore a workaround for this issue is to install the 6.0.101 SDK bundle manually by downloading it from the [.NET download site](https://dotnet.microsoft.com/download/dotnet/6.0). Once this is successfully installed scanning MU again will result in clearing the previous error.
```
Comment (mairaw) [inline release-notes/6.0/known-issues.md:22]: not related to your change but if you could apply this fix

```suggestion
Therefore a workaround for this issue is to install the 6.0.101 SDK bundle manually by downloading it from the [.NET download site](https://dotnet.microsoft.com/download/dotnet/6.0). Once this is successfully installed, scanning MU again will result in clearing the previous error.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Jay-o-Way. Changes look great. Left a couple of comments to be addressed,

File: release-notes/2.2/2.2.0/2.2.0-commits.md (hunk 1 above)
Comment (mairaw) [inline release-notes/2.2/2.2.0/2.2.0-commits.md:931]:
```suggestion
* [`[fc8979aeca]`](https://github.com/aspnet/AspNetCore/commit/fc8979aeca) Update TS docs for docs.microsoft.com generation (#2605)
```

**Review by mairaw — APPROVED** [review]

**Pull request merged**